packages/rest-explorer/package-lock.json

```diff
@@ -4,286 +4,2134 @@
 	"lockfileVersion": 1,
 	"requires": true,
 	"dependencies": {
-		"@types/body-parser": {
-			"version": "1.19.0",
-			"resolved": "https://registry.npmjs.org/@types/body-parser/-/body-parser-1.19.0.tgz",
-			"integrity": "sha512-W98JrE0j2K78swW4ukqMleo8R7h/pFETjM2DQ90MF6XK2i4LO4W3gQ71Lt4w3bfm2EvVSyWHplECvB5sK22yFQ==",
+		"@babel/code-frame": {
```

Comment:
It seems that some `@babel/` dependencies are added incorrectly. Please run `node bin/rebuild-package-locks.js @loopback/rest-explorer` to fix it.